README.md

```diff
@@ -24,7 +49,7 @@ It will accept either the individual config components:
 or just the DSN:
 
 ```erlang
-{raven, [
+{raven_erlang, [
```

Comment:
It might be a good idea to add `ipfamily` here too, since it is [required](https://github.com/artemeff/raven-erlang/blob/master/src/raven.erl#L100) in both cases.

Reply:
That's strange, I remember using it (and it was working) without any `ipfamily` entry in my configuration:
```erl
    {lager, [
        {handlers, [
            {lager_console_backend, [{level, info}]},
            {raven_lager_backend, error}
      ]}
    ]},
    {raven_erlang, [
        {dsn, "https://<uid1>:<uid2>@sentry.io/<id>"},
        {error_logger, false}
    ]}
].
```

It was working.

Was I missing something?

No problem to add it to the PR if it is indeed needed

Reply:
It will fail to match on `{ok, IpFamily}` if it's not provided - see link above to `get_config`. I tested it yesterday. Maybe it's a recent change.

Reply:
Indeed!

Added it, thx :)

Reply:
we can make it optional and use inet as default value

Reply:
:+1:  for that. This will let use the library by just specifying the DSN.

Reply:
@MonsieurV include, please :)

like `IpFamily = application:get_env(App, ipfamily, inet)` [here](https://github.com/artemeff/raven-erlang/blob/master/src/raven.erl#L100)

and then please squash commits into two — first for docs and second about ipfamily config change :)

Reply:
Ok, done it! :)

Is that good @artemeff?

I saw a default value was already provided for `ipfamily` in:

https://github.com/artemeff/raven-erlang/blob/170f036d7a8125416b3c257854735df16548c85f/src/raven_erlang.app.src#L9

I anyway added an explicit default value in the code using `application:get_env()` as you suggested.